### PR TITLE
Issue #LR-515 feat: Added the decryption tool

### PR DESCRIPTION
**Pull request by AmiableAnil**

### CODE REVIEW

File: decryption-tool/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# How to run
```
Comment (maheshkumargangula) [inline decryption-tool/README.md:1]: Please specify the technical requirements to run the tool.
Which version of Python should be installed. Will it work on Windows, Ubuntu, Mac?

File: decryption-tool/decrypt.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+   # shellcheck disable=SC2039
+   # shellcheck disable=SC2162
+   read -p 'File to decrypt: ' filename
+   # shellcheck disable=SC2162
+   # shellcheck disable=SC2039
+   read -sp 'AES key: ' aesKey
+   echo
+   python3 ./src/decrypt-l3.py "${filename}" "${aesKey}"
+else
```
Comment (maheshkumargangula) [inline decryption-tool/decrypt.sh:25]: Explicitly have a `if else` block for level 4 and else block should print invalid input.

File: decryption-tool/src/utils.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+# included in the fIn file and will be used the pyAesCrypt lib during decryption
+def decryptFileWithAESKey(filename, fIn, fOut, password):
+    try:
+        bufferSize = 64 * 1024
+        encFileSize = stat(filename).st_size
+        pyAesCrypt.decryptStream(fIn, fOut, password, bufferSize, encFileSize)
+    except ValueError:
+        # remove output file on error
+        print("Error while decryptFileWithAESKey")
+        remove("m1.txt")
```
Comment (maheshkumargangula) [inline decryption-tool/src/utils.py:64]: Is this statement required?

**Review by maheshkumargangula — CHANGES_REQUESTED** [review]

File: decryption-tool/decrypt.sh [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+# To install the dependencies
+pip3 install -r ./src/requirements.txt
+
+# Sample command to run the script
+#sh decrypt.sh
+# shellcheck disable=SC2039
+# shellcheck disable=SC2162
+read -p 'Decryption level 3 or 4: ' level
+# shellcheck disable=SC2039
+if [ "$level" == 3 ]
```
Comment (SanthoshVasabhaktula) [inline decryption-tool/decrypt.sh:11]: As suggested above remove the level understanding from the decrypt tool. Make it simple with either taking in an encryption key or pem file

**Review by SanthoshVasabhaktula — CHANGES_REQUESTED** [review]

**Review by reshmi-nair — APPROVED** [review]

**Review by reshmi-nair — APPROVED** [review]

**Review by reshmi-nair — APPROVED** [review]

**Pull request merged**